rs/consensus/src/consensus/metrics.rs

```diff
@@ -259,6 +262,12 @@ impl FinalizerMetrics {
                 // 1ms, 2ms, 5ms, ..., 10s, 20s, 50s
                 decimal_buckets(-3, 1),
             ),
+            finalization_latency: metrics_registry.histogram(
+                "consensus_finalization_latency_seconds",
+                "Wall time duration between block making and block finalization, in seconds",
+                // 1ms, 2ms, 5ms, ..., 10s, 20s, 50s
+                decimal_buckets(-3, 2),
```

Comment:
hmm, actually what I said in my comment above is not necessarily accurate (because I thought you were measuring something else); a block maker could create a block after `notary_delay` time has passed since the round start; in which case other replicas would notarize it as soon as they've received it. Still, if we take into account the latencies between nodes it should take more than a couple of milliseconds to notarize & finalize a block. Maybe, to be on the safe side, we can increase the resolution again (although I don't expect to get any observations in the lowest buckets). 

Sorry for the confusion